lib/galaxy/structured_app.py

```diff
@@ -76,6 +76,7 @@ class BasicSharedApp(Container):
     auth_manager: AuthManager
     security_agent: Any
     quota_agent: QuotaAgent
+    tool_data_tables: "ToolDataTableManager"
```

Comment:
I'd remove it from `MinimalManagerApp` below (line 116).

Reply:
Done!